api/controllers/SessionController.js

```diff
@@ -232,10 +231,12 @@ module.exports = {
     const shouldRedirect = req.method === 'GET'
     const nextUrl = req.param('n') || Frontend.Route.evo.passwordSetting()
 
-    if (req.session.authenticated) {
+    // NOTE: this was `req.session.authenticated` but that doesn't seem to
```

Comment:
@tibetsprague I need you to look at this. It works and I haven't been able to find another way to make it work after going down the Passport rabbit hole a ways. Just needs extra eyes. I've tested and am not seeing any issue.